### DISCUSSION TITLE
Log titer conversion

### DISCUSSION THREAD

**Discussion by SCUZhoull** [acorg/Racmacs#172] [Q&A]
I followed the steps in "Making an antigenic map from titer data" to generate an antigenic map using the example data "extdata/h3map2004_hitable.csv". I got a map which is similar to the map in this introduction. I just wonder, do the steps in "Making an antigenic map from titer data" already convert the titer in "extdata/h3map2004_hitable.csv" onto the log scale? Or do I need to do extra steps of log titer conversion and then using acmap(map) and optimizeMap(map) to generate an antigenic map? 

And I also wonder execpt for the transformation $log2(x/10)$, is there anyother option for titer transformation? I mean can I choose other menthods for titer transformation?

Thanks for your attention and I am looking forward to your reply.

**Comment by drserajames** [comment]
You don't need to manually do the conversion to log titre, it is part of the map-making functions.

I don't know if how easy it is to use another transformation, but I expect it would be tricky. @shwilks would know. Did you have a particular transformation in mind?

**Comment by SCUZhoull** [comment]
Thanks very much for your reply! I do not have a particluar transformation yet. I used ELISA or neutralizing antibody titers to generate antigenic maps. I am not sure whether these data are suitable for the transformation log2(x/10). So I just wonder whether there is any other way of data transformation. Thanks a lot!

**Comment by drserajames** [comment]
I use Racmacs for different neutralisation assays and it works fine. I don't know for ELISA; if your result is analogous to a concentration then I would expect the current transformation to be appropriate. But please do open another discussion if you try and get some weird looking maps!
